app/views/spree/admin/subscriptions/failures.html.erb

```diff
@@ -1,31 +1,52 @@
 <% content_for :page_title do %>
-  Failed Renewals
+  <%= Spree.t(:'admin.subscription.failures') %>
 <% end %>
 
-<b> Total: <%= @subscriptions.count %>
-<br/>
-<br/>
+<% unless @subscriptions.empty? %>
+<div class="pagination">
+  <%= page_entries_info @collection %>
+</div>
 <table id="listing_subscriptions" class="index responsive">
   <thead>
     <tr>
-      <th>id</th>
-      <th>date</th>
-      <th>order</th>
-      <th>email</th>
-      <th>renewal date</th>
-      <th>attempts</th>
+      <th>ID</th>
+      <th>Status</th>      
+      <th>Date</th>
+      <th>Order</th>
+      <th>Renew Date</th>
+      <th>Attempts</th>
     </tr>
   </thead>
   <tbody>
     <% @subscriptions.each do |subscription| %>
       <tr>
-        <td><%= link_to subscription.id, edit_admin_subscription_path(subscription), target: '_blank' %>
-        <td><%= l(subscription.last_order_date, format: :subscription_date_format) %>
-        <td><%= link_to subscription.orders.first.number, edit_admin_order_path(subscription.orders.first), target: '_blank' %>
-        <td><%=link_to subscription.user.email, edit_admin_user_path(subscription.user), target: '_blank' %>
-        <td><%= l(subscription.next_shipment_date, format: :subscription_date_format) %>
-        <td><%= subscription.failure_count %>
+        <td><%= link_to subscription.id, edit_admin_subscription_path(subscription), target: '_blank' %></td>
```

Comment:
Does it make sense to add to the feature tests for some of these? Not completely sure here since the index page is tested at the moment.

Reply:
probably should @braidn 

my idea is to get this out to CX so they can start addressing the issues. I have opened an RFC to further improve this where things like UI and such may change but definitely the improvements will come with specs. I'm just afraid things may continue to change from here.